pkgs/applications/networking/p2p/tribler/default.nix

```diff
@@ -1,48 +1,50 @@
-{ stdenv, fetchurl, pythonPackages, makeWrapper, imagemagick
-, enablePlayer ? true, vlc ? null, qt5 }:
+{ stdenv, fetchurl, pkgs, python3Packages, makeWrapper, imagemagick
+, enablePlayer ? true, vlc ? null, qt5, lib }:
 
 stdenv.mkDerivation rec {
   pname = "tribler";
-  version = "7.1.2";
+  version = "7.4.0-exp1";
 
   src = fetchurl {
-    url = "https://github.com/Tribler/tribler/releases/download/v${version}/Tribler-v${version}.tar.gz";
-    sha256 = "1ayzqx4358qlx56hsnsn5s8xl6mzdb6nw4kwsalmp86dw6vmmis8";
+    url = "https://github.com/Tribler/tribler/releases/download/v${version}/Tribler-v${version}.tar.xz";
+    sha256 = "18ziisg0v2gdxnprbhqsryz92yk270waj0la7m2h326k5qql3qkf";
   };
 
   buildInputs = [
-    pythonPackages.python
-    pythonPackages.wrapPython
+    python3Packages.python
+    python3Packages.wrapPython
     makeWrapper
```

Comment:
`wrapPython` and `makeWrapper` should be `nativeBuildInputs` - not sure about imagemagick - where is that required?

Reply:
Right. I'll double check the whole expression and deps - I was mostly focused on getting python 3 to work.

Reply:
Thanks :-)

Reply:
imagemagick: was used to strip ICC profiles from UI graphics that used to cause issues: https://github.com/Tribler/tribler/pull/1561. I ditched that now and also moved those into nativeBuildInputs.